lowpass_filter_design/README.md

```diff
@@ -0,0 +1,30 @@
+# Lowpass Filter Generation
+
+This code generates the lowpass filter to avoid aliasing for audio downsampled to 16kHz. Therefore the cutoff frequency needs to be slightly larger than 8 kHz.
+
+## Quickstart
+
+Generate this filter, or design a new filter
+```
+pipenv sync
+pipenv run python generate_filter.py
+```
+
+This code will print the filter's coefficients and save an image of its frequency response. To update the filter, copy the printed coefficients to `src/Microphone.js`.
+
+## Design
+
+The filter designed is a FIR type 1 with with linear phase*. The filter's cutoff frequency is 9.702kHz for a sampling rate of 44.1kHz. The filter order chosen was 41 (or 41 coefficients) because the transition bandh (magnitude response drop off) was deemed acceptable (by Colin) while being a relatively low number for computational efficiency. The cuttoff frqeuency is conservativly high to avoid attenuating energy from 0 to 8 kHz.
+
+The filters frequency response is shown below.
+
+![Lowpass Filter Frequency Response](filter_frequency_response.png)
+
+## Implementation Notes
+If the input sampling rate is greater than 44.1kHz the cuttoff frequency will increase.
```

Comment:
```suggestion
If the input sampling rate is greater than 44.1kHz the cutoff frequency will increase.
```

Reply:
Fixed in ddfbcfa5fa241632362249f7bf4b7a5c94ae2d21